cicecore/cicedynB/infrastructure/ice_restart_driver.F90

```diff
@@ -58,9 +58,7 @@ subroutine dumpfile(filename_spec)
           stressp_1, stressp_2, stressp_3, stressp_4, &
           stressm_1, stressm_2, stressm_3, stressm_4, &
           stress12_1, stress12_2, stress12_3, stress12_4
-#ifdef CESMCOUPLED
```

Comment:
This still needs CESMCOUPLED around it as it is not used in the standalone CICE. Is there an equivalent ifdef for NEMS?